aries-site/src/examples/components/namevaluelist/Examples/components/NameValueListFormLabel.js

```diff
@@ -9,10 +9,16 @@ import { Box, Text } from 'grommet';
 export const NameValueListFormLabel = ({ data, name }) => {
   const theme = useContext(ThemeContext);
   return (
-    <Box margin={{ top: 'small' }} >
+    <Box margin={{ top: 'small' }}>
       <Text as="label" {...theme.nameValuePair.name} htmlFor={name} name={name}>
         {data.displayName}
-        {data.required ? <Text aria-label="required" weight="normal">*</Text> : undefined}
+        {data.required ? (
+          <Text aria-label="required" weight="normal">
+            *
+          </Text>
+        ) : (
+          undefined
+        )}
```

Comment:
Would this achieve the same result?

```suggestion
        {data.required && (
          <Text aria-label="required" weight="normal">
            *
          </Text>
        )}
```

Reply:
Yup that works, including in next commit.